ykllvmwrap/src/jitbuilder.cc

```diff
@@ -0,0 +1,306 @@
+using namespace llvm;
```

Comment:
Can we have a comment at the top of the file explaining what this does.

Reply:
Fixed in 9f14d25.